docs/shaka/results/issues.md

```diff
@@ -0,0 +1,58 @@
+## Issue #1
+
+HLS wiht Widevine is not suported. It may be thatAzure  is not providing the EXT-X-Key in the HLS manifest.
```

Comment:
Typos "with"

Reply:
Typso "that Azure"

Reply:
Maybe state this as Azure Media Services does not yet support EXT-X-KEY in the manifest for Widevine with HLS. This features is currently on the backlog